src/main/java/de/jutzig/github/release/plugin/UploadMojo.java

```diff
@@ -189,6 +189,7 @@ public void execute() throws MojoExecutionException {
 			if (release == null) {
 				getLog().info("Creating release "+releaseName);
 				GHReleaseBuilder builder = repository.createRelease(tag);
+				builder.generateReleaseNotes(true);
```

Comment:
I was on the fence between having it in the `else` branch (that is, when description is null) and always adding it, and decided that GitHub is doing a better job in most of the cases. Any customization still be added on top as specified in the previous comment.